fpga-arch/bsv/Prescaler.bsv

```diff
@@ -0,0 +1,56 @@
+// This file is part of Scaffold
+//
+// Scaffold is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+//
+// Copyright 2024 Ledger SAS, written by Charles Christen
+
+package Prescaler;
+
+import Counter::*;
+
+(* always_ready, always_enabled *)
```

Comment:
According to the reference guide (https://csg.csail.mit.edu/6.S078/6_S078_2012_www/resources/reference-guide.pdf), `always_enabled` implies `always_ready`, so I think for this declaration and the others, only `always_enabled` is enough.